array_family/flex_ext.py

```diff
@@ -469,6 +471,48 @@ def subsort(self, key0, key1, reverse=False):
             ref_tmp.sort(key1, reverse)
             self[min(val) : (max(val) + 1)] = ref_tmp
 
+    def match_by_hkle(
+        self, other: dials_array_family_flex_ext.reflection_table
+    ) -> Tuple[cctbx.array_family.flex.size_t, cctbx.array_family.flex.size_t,]:
```

Comment:
Somehow, this dropped out of my collection of suggested changes.
```suggestion

```    ) -> Tuple[2 * (cctbx.array_family.flex.size_t,)]:

Reply:
An alternative, possibly more readable, option would be to declare a [type alias](https://docs.python.org/3/library/typing.html#type-aliases), e.g.:

```
FlexSizeT = cctbx.array_family.flex.size_t
...
def match_by_hkle(...) -> Tuple[FlexSizeT, FlexSizeT]
```

Reply:
Indeed.